cg/meta/clean/demultiplexed_flow_cells.py

```diff
@@ -257,13 +257,14 @@ def archive_sample_sheet(self):
         globbed_unaligned_paths: Path.glob = Path(self.path).glob(
             DemultiplexingDirsAndFiles.UNALIGNED_DIR_NAME + ASTERISK
         )
-        if not list(globbed_unaligned_paths):
+        globbed_unaligned_paths_list: list = list(globbed_unaligned_paths)
+        if not globbed_unaligned_paths_list:
             LOG.warning(
                 "No Unaligned directory found for flow cell %s! No sample sheet to archive!",
                 self.run_name,
             )
             return
-        unaligned_path = list(globbed_unaligned_paths)[0]
+        unaligned_path: Path = list(globbed_unaligned_paths_list)[0]
```

Comment:
```suggestion
        unaligned_path: Path = globbed_unaligned_paths_list[0]
```
I guess it is already so much a list as you can make it ? :)